doc/source/known-issues.rst

```diff
@@ -1,97 +1,126 @@
+.. index:: Known Issues
+.. _Known Issues:
+
 Known Issues
 ============
 
-This is a short list of known issues.
+This section provides a short list of known issues.
+
+.. index:: Known Issues, Mount Path Limit
+.. _Mount Path Limit:
 
-88 character mount path limitation
-----------------------------------
+Mount Path Limit
+----------------
 
-There is a know mountpoint path length limitation issue on FreeBSD which is set to a historical 88 character limit.
+There is a known mountpoint path length limitation issue on FreeBSD.
+Path length has an historical 88 character limit.
 
-This issue does not affect iocage jails from functioning properly, but can present challenges
-when diving into ZFS snapshots (cd into .zfs/snapshots, tar, etc.).
+This issue does not affect :command:`iocage` jails from functioning
+properly, but can present challenges when diving into ZFS snapshots,
+like :command:`cd` into :file:`.zfs/snapshots`, :command:`tar`, etc.
 
 ZFS snapshot creation and rollback is not affected.
 
-To workaround this issue iocage 1.6.0 introduced a ``hack88`` property.
+To work around this issue, **iocage 1.6.0** introduced a **hack88**
```

Comment:
`hack88` no longer exists. We suggest the user either supplies their own `--name`, or use `--short` if that functionality is needed.